lib/ecto/adapters/sql.ex

```diff
@@ -246,28 +246,63 @@ defmodule Ecto.Adapters.SQL do
   end
 
   @doc """
```

Comment:
I'm planning to improve docs as soon other adapters are implemented (TBD in this PR)